sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -359,6 +359,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val THRIFTSERVER_RESULT_LIMIT =
+    buildConf("spark.sql.thriftserver.retainedResults")
```

Comment:
How about Hive thrift server? Does it has the similar parameters? The parameter name does not look straightforward to me.